docs/examples/notebooks/multichannel-coupled-histo.ipynb

```diff
@@ -16,6 +16,7 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
+      "%pylab is deprecated, use %matplotlib inline and import the required libraries.\n",
```

Comment:
Could consider replacing `%pylab inline` by
```python
import numpy as np
import matplotlib.pyplot as plt
```
if you are touching this file already.

Reply:
I'll leave that up to @matthewfeickert to go through all notebooks and do this kind of update.

Reply:
@kratsg yeah I can do that. We should also try to figure out a way to have these notebooks get moved to something that is either outside of the main repo or built out each time as I think that most of the size of the GitHub repo comes from the image diffs in notebooks, which feels bad. :grimacing: